src/App.js

```diff
@@ -1,23 +1,45 @@
-import logo from './logo.svg';
 import './App.css';
+import React, {useState} from 'react';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFoodForm from './components/AddFoodForm';
+import Search from './components/Search';
+
 
 function App() {
+  const [dishes, setDishes] = useState(foods)
+  const [showDishes, setShowDishes] = useState(foods);
+
+  const createFood = (newFood) => {
+    const updatedFood = [newFood, ...dishes]
+    setDishes(updatedFood)
+  }
+
+  const searchFood = (searchQuery) => {
+    const filteredFood = dishes.filter((food) => 
+    food.name.toLowerCase().includes(searchQuery.toLowerCase())
+    );
+    setDishes(filteredFood);
```

Comment:
Here you want to update only the 'showDishes' not the 'dishes' so instead of:
`setDishes(filteredFood);`
Do:
`setShowDishes(filteredFood);`